framework/src/main/java/org/tron/core/net/TronNetDelegate.java

```diff
@@ -206,6 +211,9 @@ public void processBlock(BlockCapsule block) throws P2pException {
             backupServerStartFlag = true;
             backupServer.initServer();
           }
+
+          //record metrics
+          metricsService.applyBlock(block);
```

Comment:
should handle before dbManager.pushBlock(block);

Otherwise, there is a problem in the statistics of latency